.travis.yml

```diff
@@ -38,11 +38,11 @@ before_install:
 
 install:
   - pip install --upgrade pip setuptools wheel
-  - pip install --only-binary=numpy,scipy,matplotlib numpy scipy matplotlib
-  - pip install future gitpython pytidylib
+  - pip install --only-binary=matplotlib,numpy,scipy matplotlib numpy scipy
```

Comment:
Why is this needed for your issue? If it is not related to your issue, please make such changes on a new issue and branch.